client/src/components/Feed/Post.js

```diff
@@ -346,7 +362,7 @@ const Post = ({
           }}
         >
           <div className="card-header">
-            {renderHeader}
+            {renderHeader()}
```

Comment:
Personally I don't really think creating separate functions for parts of the rendered output makes much sense 
this is usually a good idea for things like mapped functions, like `items.map(renderItem)` but otherwise I don't really see an advantage to create multiple separate functions unless they are extracted to another file

Reply:
Are you speaking more general - all the `renderSocialIcons`, `renderShareModel`, `renderHeader` etc and not just my specific change? So basically - they should be moved to a separate component file or just include within the main structure.

Anyways I thought I had made a change to address this for my specific function but I now I realize probably not.